src/docs/truffle/getting-started/debugging-your-contracts.md

```diff
@@ -197,6 +197,12 @@ This command can also print locations other than the stack, if you want to view
 
 You can also add these extra locations to the default display with `+`; e.g., `p +mem` will make it so that memory will always be displayed when you enter `p` or `;`, and `p -mem` will turn this off.  You can even turn off the stack display with `p -sta`, or force it to display with `p sta`.  All of these options can again be combined.
 
+This command prints a group of instructions surrounding the current instruction. By default, it prints 3 instructions ahead and 3 instructions back of the current instruction.  You can configure the number of instructions to be printed with `+` and `-`; e.g., `p +<instructions-ahead> -<instruction-back>` will make it to print `instructions-ahead` instructions ahead and `instructions-back` instructions back of current instruction. The configured numbers are saved and used for printout afterward.
```

Comment:
Editing suggestions:

1. "This command prints" -> "This command also prints"
2. "make it to print" -> "make it print" or "cause it to print"
3. "`instructions-ahead` instructions ahead and `instructions-back` instructions back of current instruction" -> "`instructions-ahead` instructions ahead of and `instructions-back` instructions behind the current instruction"